EventListener/UserContextSubscriber.php

```diff
@@ -146,18 +153,24 @@ public function onKernelResponse(FilterResponseEvent $event)
                 return;
```

Comment:
@dbu OK, that's fine by me, and the sanity check looks like a very nice improvement !